.github/workflows/i18nCheck.yml

```
@@ -0,0 +1,14 @@
+on: 
+  push:
+    paths: ['**.R', '**.qml', 'tests/**', 'inst/**', '**.Rd', '**.c', '**.cpp', '**.h', '**.hpp', '**.yml']
+ 
+  pull_request:
+    paths: ['**.R', '**.qml', 'tests/**', 'inst/**', '**.Rd', '**.c', '**.cpp', '**.h', '**.hpp', '**.yml']
```

```suggestion
    paths: ['**.R', '**.qml', '**.yml']
 
  pull_request:
    paths: ['**.R', '**.qml', '**.yml']
```
is there any reason to trigger this for other file types? As far as I know, only R and qml are translated.

---

yes, you are right.